### PR TITLE
Added functions to get significand and exponent of floating point sorts

### PR DESCRIPTION
**Pull request by DavidThien**
As title says. Pretty small change to add some functions I needed.

### CODE REVIEW

File: z3/src/sort.rs [hunk 1]
```diff
@@ -154,6 +154,22 @@ impl<'ctx> Sort<'ctx> {
         unsafe { Z3_get_sort_kind(self.ctx.z3_ctx, self.z3_sort) }
     }
 
+    pub fn get_float_exponent_size(&self) -> Option<u32> {
```
Comment (fitzgen) [inline z3/src/sort.rs:157]: Please remove the `get_` prefix, as it is generally unidiomatic Rust (this crate isn't fully consistent with this practice, but gating new changes on being idiomatic helps move us in the right direction).

**Review by fitzgen — COMMENTED** [review]
Looks great! Before I merge this, would you mind adding documentation comments for each method describing what they do, the units of the returned value, when `None` is returned, etc? Thanks!

**Comment by DavidThien** [comment]
Added documentation and updated the function names

**Review by fitzgen — COMMENTED** [review]
Oh sorry, I forgot to mention this last time, but one more thing: can you make a smoke test in `tests/lib.rs` double checking that we get the expected significand and exponent sizes for a float? And `None` for an integer or some other non-float type?

Thanks!!

**Comment by DavidThien** [comment]
Np! Added a test as well

**Review by fitzgen — APPROVED** [review]
Thanks!

**Pull request merged**